ArduPlane/Parameters.h

```diff
@@ -146,6 +146,8 @@ class Parameters {
         k_param_parachute,
         k_param_arming = 100,
         k_param_parachute_channel,
+        k_param_parachute_auto_enabled,
```

Comment:
the order here is extremely important. You can't push down other k_params indexes. You must use an unused index